packages/frontend/src/creation-form/components/rewards/table/row.tsx

```diff
@@ -154,14 +154,12 @@ export const RewardRow = ({
                         root: "text-center",
                     }}
                 >
-                    {!noFees ? formattedAmountAfterFees : formattedAmount}
+                    {formattedAmount}
```

Comment:
Why remove the condition for the `!noFee` here?

Reply:
I initially removed this because it's a bit confusing otherwise. In the case in which the user has picked 10 TKN as the reward amount, he'd see 10.1 TKN in the rewards table while the end amount that gets into the campaign is actually 10 TKN. 

Now that I think about it though, we could just pass `noFees` while the table is in the rewards picking step and have the rewards table displayed in the deploy step include the fees, wdyt?

Reply:
Fixed in c1766f5a9d53a19f9efb942043eea471db54164c

Reply:
Exactly. I should have fixed that and a couple other things in https://github.com/carrot-kpi/erc20-kpi-token-template/pull/207/commits/480314b87860b012b037ed92895d08d9109e096e